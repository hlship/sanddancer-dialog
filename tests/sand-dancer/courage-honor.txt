%% sand-dancers arrival needs one complete plan, or two in-progress plans
(grab [#courage #duct-tape #luck #canned-oranges])
(setup sand-dancers offer)
u
say hello
ask who
%% Trigger missing plan items
ask next
l
take gas can
say not sure
ask honor
trade job for honor
trade last day for honor
trade road trips for honor
i



