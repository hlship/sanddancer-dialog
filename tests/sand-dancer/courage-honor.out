

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (grab [#courage #duct-tape #luck #canned-oranges])
Query succeeded: (grab [#courage #duct-tape #luck #canned-oranges])
> (setup sand-dancers offer)

Storage Room
Waves of sand flow like water through the room, filling it higher and higher.
You glance wildly at the exits south and up.

Query succeeded: (setup sand-dancers offer)
> u
Barely able to reach the ladder through sand up to your waist, you haul yourself
up the rungs and pull free just in time.

Roof
Except it's more like a foundation now, the building's sunk almost entirely into
the sand.

The sand swirls and churns like something liquid or alive around you, and then
you look and see it is alive, thousands and thousands of lizards, little sand-
dancers, are burrowing up from somewhere underground, scattering and writhing in
some complex pattern.

Then, suddenly, everything stops.

And with a mighty heave, a huge lizard crests through the sand right in front of
you.

The lizard's tongue snakes through the air, tasting it, tasting you. One slitted
eye studies you intently. It pauses, as if waiting for you to say something.

You could say hello.

> say hello
Stunned by what you see, you stammer out a greeting.

The ground rumbles as the lizards shift. "Hello," he says and his voice shakes
the earth and creaks the rusty joins of the tower. "Yes, hello, little one.
Strange thing to say when I've always been with you, watching, waiting. But
hello, hello at last, Nakaibito."

You could ask who he is or ask about the lizards.

> ask who
"Who, uh, what are you?" you stammer.

"I am Sand-dancer," he breathes flicking his toungue in and out, slowly. "I am
the desert and the dust and the rain and the wind. I was there the first time
you went hungry and the first time you made love and the last time you asked for
help, but I could not help you then. But tonight, Nakaibito, tonight I can."

You could ask what to do next or ask about the lizards.

> ask next
"I've been kicking around here all night, seems like I'm no closer to leaving
than when I started" you say.

"Much has happened tonight. You've made bargains and plans. You hold courage
that earned you a tool for mending. You wear luck that filled your belly. But
these things together do not help you. No, you still haven't decided." The
lizard stares at you with one eye, his great slittled eye blinking. "But now is
the time."

The ground shudders and shifts beneath you, and out of the sand rises the
emergency blanket and the gas can.

"The tape and gas can will return you to her. The blanket and the can of food
will look out for you. Take one."

> l
Roof
Except it's more like a foundation now, the building's sunk almost entirely into
the sand.

Thousands of lizards writhe in the desert around you. Sand-dancer sits before
you, breathing, watching.

On the sand before him are a gas can and an emergency blanket.

Something about the gas can makes you want to brood about road trips through the
desert.

> take gas can
You take the gas can. As you do, the emergency blanket sinks into the sand.

"By your actions and through your talents, you have chosen to repair your
vehicle and return to her" he says, sighing as the lizards in the desert writhe
and twitch. "You have looked into yourself and found your story, the path you
will walk tomorrow."

He breathes. "And are you happy with your story?" he asks, and there's some
dangerous undertone to the question and you don't know why. The lizards tense,
quivering, twisting their little heads back and forth as they await your answer.

You could say yes, say no, or say you're not sure.

> say not sure
You open your mouth to say something and... no words come out. You don't know if
you've made the right choice or not. You stare helplessly at the giant lizard.

The lizard turns his head sharply, and all the lizards in the desert around
writhe angrily, clawing and biting at each other in agitation. "And when will
you know, Knock?" the lizard asks. "When will you own your life? When will you
become a man? Tonight. It must be tonight."

The little sand-dancers writher and chitter all around you.

A glistening egg rises from the sand, glowing with neon potential, the lizard
glances down at it almost dismissively. "This is honor," he says, "the last
talent you need to achieve your goal, and you can have it for three memories.
But perhaps I have something else to trade."

The lizards whisper in anticipation as a second egg rises. This one's bigger,
and almost blinding with its intensity, and the lizards draw back, chittering in
awe and amazement. "This," Sand-dancer says, "is freedom. Pure, undiluted, and
absolute. Many crave it and few get a chance to take it. I offer it to you,
Nakaibito, but it does not come cheaply." He grins. "For freedom, you must give
up everything. You must give me all your memories."

The lizards begin to circle restlessly as they wait for you to choose. Sand-
dancer merely watches, and waits.

You could ask about honor or ask about freedom.

> ask honor
"What's honor got to do with getting out of this dump?" you ask.

"You have the tools to fix your truck," he says, "but to return to Ocean and
stand by her side you will need honor."

You could ask about freedom.

> trade job for honor
He flicks out his tongue and pulls your shit job from your head, swallowing it
whole. "A good start," he says, "but you must trade more."

You could ask about freedom.

> trade last day for honor
He flicks out his tongue and pulls your last day of high school from your heart,
swallowing it whole. "Yes," he says, "but more."

You could ask about freedom.

> trade road trips for honor
Sand-dancer nods. "Yes," he says, "a fair trade." And something happens inside
you as he says it. The memory of road trips through the desert shifts and
wriggles and fades away and it's still there, but now it's shifted, become
something else. And, yeah, it kinda does feel like you could call it honor.
Cool. "The choice is made," he rumbles, "your path is set. Goodbye, Nakaibito."
And he turns and dives into the sand. The lizards twist and writhe and follow
him.

And then, just as quiet settles, another huge rumble shakes the ground, and,
almost like an afterthought, the building breaches the sand and rises to the
surface.

Crumbling Concrete
This concrete building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> i
You have a gas can, a flashlight, some canned oranges, a roll of duct-tape, your
lighter, your wallet, and your emotional baggage. You're wearing your denim
jacket. You've gained the talents courage, luck, and honor.

In your emotional baggage are watching Family Guy with Karl, meeting Ocean, and
your grandma's stories.

> s
You go, and the clouds are gone and the stars are so bright you squint. You
patch the fuel line and refill the tank and she starts like a beauty, and then
you're back on the highway again, headed for Ocean, and you've never felt
luckier.

You shut off the headlights as you pull up to the trailer, and climb in through
her window for old times sake, crazy courage her sister at least appreciated if
not her old man. She's awake of course, always no matter how quiet you are, and
she holds out a hand from somewhere under the blankets. And as she's doing it
you're suddenly exhausted, beyond exhausted but also safe, warm even before you
slip under the covers. She half turns to kiss you as you fumble with the
blankets but you're so tired all you can do is push your face against her neck
and slip an arm around her warm skin, and she starts to ask you something but
it's too late, you're crashing, you're crashing, you crash...

     *** You have honor. ***

Would you like to:
     UNDO the last move,
     RESTORE a saved position,
     QUIT the program,
     or RESTART from the beginning?
> 
